WebContent/js/actions/windowTitle.js

```diff
@@ -0,0 +1,37 @@
+import { DEFAULT_TITLE } from '../reducers/windowTitle';
+
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2016, 2019
```

Comment:
Copyright year should only be for 2019 as this file was only created and updated in 2019.